lib/generators/manageiq/provider/USAGE

```diff
@@ -5,3 +5,4 @@ Example:
     rails generate manageiq:provider ManageIQ::Providers::VendorName
     rails generate manageiq:provider ManageIQ::Providers::VendorName --path ~/dev
     rails generate manageiq:provider ManageIQ::Providers::VendorName --vcr --dummy
+    rails generate manageiq:provider ManageIQ::Providers::VendorName --vcr --dummy --manager-type=cloud
```

Comment:
Now that it's required, we need to update the other USAGEs

Reply:
It is actually only required if you pass `--dummy` otherwise the manager classes aren't created

Reply:
Right, so then at least the line before it is invalid?

Reply:
Yes meant to delete that not leave it

Reply:
Oh, interesting....if `--dummy` means scaffold, and the value is required when dummy is passed, then maybe we just drop dummy altogether and merge it with manager-type...perhaps  `--scaffold=cloud` or `--scaffold-manager-type=cloud`

Reply:
Yeah it also seems weird that `--no-dummy` is the default, because this is almost the same as the `plugin` generator except for creating a `empty_directory "spec/models/#{plugin_path}"` directory.

I wonder if we change it from defaulting to not creating any of those, to creating them by default and having an option `--no-scaffolding` to disable it if you want?

Reply:
I like that.

Reply:
:+1: done